api_poll/api-server-poll.yml

```diff
@@ -14,7 +14,7 @@
     description: |
         Managed by Jenkins Job Builder, do not edit manually!
     triggers:
-      - timed: "H * * * *"
+      - timed: "*/30 * * * *"
```

Comment:
this will run each 2 min. this can be done `H/30` as well. this makes jobs get distributed over time.